style-dictionary/actions/concat-files.mjs

```diff
@@ -1,60 +1,60 @@
-import fs from 'fs-extra';
-import concat from 'concat';
-import path from 'path';
-import { getDirname } from '../utils.mjs';
+import fs from 'fs-extra'
```

Comment:
Was this file changed? I would like to get Prettier in here, which would alleviate these types of questions, but we have not added it in yet ☹️

Reply:
I run eslint to fix those problems. But yeah, I agree. We need Prettier. Do you want me to add it in this PR too?

Reply:
Lets get in the next one after this after merged in, it's ok as-is